srcpkgs/hurl/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'hurl'
+pkgname=hurl
+version=1.0.0
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="libxml2-devel libressl-devel"
+short_desc="Hurl, run and test HTTP requests. "
+maintainer="Juan Tascon <juan@horlux.org>"
+license="Apache-2.0"
+homepage="https://github.com/Orange-OpenSource/hurl"
+changelog="https://github.com/Orange-OpenSource/hurl/raw/master/CHANGELOG.md"
+distfiles="https://github.com/Orange-OpenSource/hurl/archive/${version}.tar.gz>${pkgname}-${version}.tar.gz"
+checksum=3c1bbd3650a91d8434f9a57b0123246c3260e955a966feb2c63dedba94463ac1
+
+do_build() {
+	./ci/man.sh
```

Comment:
This makes me think its would only generate the man pages and not actually build the binaries?